src/test/java/org/libgit2/jagged/BlobTest.java

```diff
@@ -77,14 +77,7 @@ public void testGetFilteredContent()
     {
         byte[] expected;
 
-        if (Platform.getCurrentPlatform().getOperatingSystem().equals(OperatingSystem.WINDOWS))
-        {
-            expected = "This is file two!\r\n".getBytes();
-        }
-        else
-        {
-            expected = "This is file two!\n".getBytes();
-        }
+        expected = "This is file two!\n".getBytes();
```

Comment:
This suggests that our tests are not properly sandboxing their environment.  There should be a `.gitattributes` that induces the correct CRLF behavior here.  If there's not, then we should add that.  This just changes the breakage to happen to work in your environment and break in most other Windows users.